src/d3a/models/area/event_types.py

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
Please add license header here.

Reply:
Adding the header, same goes for the following comment.